assets/svelte/components/SelectedElementFloatingMenu.svelte

```diff
@@ -0,0 +1,258 @@
+<script lang="ts" context="module">
```

Comment:
This file is called "selectedElementFloatingMenu", but it doesn't seem like a generic floating menu because it has all the logic for reordering. That's ok and we can make a generic one later if you'd like, but in that case I think we should rename this one.

Reply:
True, it was meant to be a floating menu but it keep getting more and more logic for the drag and drop. I should move most of this logic out into a drag button of sorts